wpilib/wpilib/command/commandgroup.py

```diff
@@ -6,11 +6,11 @@
 # the project.
 #----------------------------------------------------------------------------
 
-from .command import Command
+from . import command
 
 __all__ = ["CommandGroup"]
 
-class CommandGroup(Command):
+class CommandGroup(command.Command):
```

Comment:
Why?

Reply:
Circular imports. Command-based has a lot of circular dependency which doesn't work well with python. To include the type hints and keep the program working, I had to use a lot of module importing

Reply:
There's no circular import here at runtime.